clang/test/Driver/aie2/aie2-stdlib.c

```diff
@@ -0,0 +1,25 @@
+//===- aie2-stdlib.c ------------------------------------------*- C++ -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+//
+//===----------------------------------------------------------------------===//
+
+// RUN: %clang %s -### --target=aie2-none-unknown-elf -ccc-install-dir  %S/../Inputs/basic_aie_tree/bin 2>&1 \
```

Comment:
I think your proposal in https://github.com/Xilinx/llvm-aie/pull/55#discussion_r1628791567 makes sense.
Could you apply the change?